101-application-gateway-rewrite/azuredeploy.json

```diff
@@ -64,77 +53,21 @@
     }
   },
   "variables": {
-    "applicationGatewayName": "[parameters('applicationGatewayName')]",
-    "idName": "identity1",
-    "keyVaultName": "[concat(uniqueString(resourceGroup().name), '-kv')]",
-    "publicIPAddressName": "[concat(resourceGroup().name, '-publicIp')]",
-    "virtualNetworkName": "[concat(resourceGroup().name, '-vnet')]",
-    "identityID": "[resourceId('Microsoft.ManagedIdentity/userAssignedIdentities',variables('idName'))]"
+    "virtualNetworkName": "[concat(resourceGroup().name, '-vnet')]"
   },
   "resources": [
-    {
-      "type": "Microsoft.ManagedIdentity/userAssignedIdentities",
-      "apiVersion": "2015-08-31-PREVIEW",
-      "name": "[variables('idName')]",
-      "location": "[parameters('location')]"
-    },
-    {
-      "type": "Microsoft.KeyVault/vaults",
-      "apiVersion": "2018-02-14",
-      "name": "[variables('keyVaultName')]",
-      "location": "[parameters('location')]",
-      "dependsOn": [
-        "[concat('Microsoft.ManagedIdentity/userAssignedIdentities/', variables('idName'))]"
-      ],
-      "properties": {
-        "sku": {
-          "name": "standard",
-          "family": "A"
-        },
-        "tenantId": "[reference(variables('identityID'), '2015-08-31-PREVIEW').tenantId]",
-        "accessPolicies": [
-          {
-            "tenantId": "[reference(variables('identityID'), '2015-08-31-PREVIEW').tenantId]",
-            "objectId": "[reference(variables('identityID'), '2015-08-31-PREVIEW').principalId]",
-            "permissions": {
-              "secrets": [
-                "get"
-              ]
-            }
-          }
-        ],
-        "enableSoftDelete": true
-      }
-    },
-    {
-      "type": "Microsoft.KeyVault/vaults/secrets",
-      "apiVersion": "2018-02-14",
-      "name": "[concat(variables('keyVaultName'), '/', parameters('Secrets')[copyIndex()].name)]",
-      "location": "[parameters('location')]",
-      "dependsOn": [
-        "[resourceId('Microsoft.KeyVault/vaults', variables('keyVaultName'))]"
-      ],
-      "copy": {
-        "name": "secretcopy",
-        "count": "[length(parameters('Secrets'))]"
-      },
-      "properties": {
-        "value": "[parameters('Secrets')[copyIndex()].value]",
-        "recoveryLevel": "Purgeable"
-      }
-    },
     {
       "type": "Microsoft.Network/publicIPAddresses",
       "apiVersion": "2018-10-01",
-      "name": "[variables('publicIPAddressName')]",
+      "name": "[parameters('publicIPAddressName')]",
       "location": "[parameters('location')]",
       "sku": {
         "name": "Standard"
       },
       "properties": {
         "publicIPAllocationMethod": "Static",
         "dnsSettings": {
-            "domainNameLabel": "[uniqueString(resourceGroup().name)]"
+            "domainNameLabel": "[concat('domain', uniqueString(resourceGroup().name, parameters('applicationGatewayName')))]"
```

Comment:
use the parameter you have defined for this

Reply:
@bmoore-msft using the parameter now